core/lib/presentation/utils/style_utils.dart

```diff
@@ -10,9 +9,6 @@ class CommonTextStyle {
     fontWeight: FontWeight.normal,
   );
 
-  static const defaultTextOverFlow = PlatformInfo.isWeb
-      ? TextOverflow.fade
-      : TextOverflow.ellipsis;
-
-  static const defaultSoftWrap = PlatformInfo.isWeb ? false : true;
+  static const defaultTextOverFlow = TextOverflow.ellipsis;
```

Comment:
I dont remember why we should use fade, please carefully

Reply:
We use it because it crashes the font `Inter` on the web. But we shouldn't change it at this time, so I flagged checking `PlatformInfo.isWebMobile`

Reply:
Avoid `Inter` font error when using `TextOverflow.ellipsis` on earlier versions of flutter 2.10. But now on 3.16 it is working fine.